.github/workflows/development-environment.yml

```diff
@@ -42,16 +43,17 @@ jobs:
         # Sometimes, brew needs to be updated before brew bundle would work
         # After installing Docker (via homebrew) we need to make sure that it is properly initialized on Mac
         run: |
-          HOMEBREW_NO_AUTO_UPDATE=1 brew bundle -q --no-upgrade || brew update && brew bundle
-          SENTRY_NO_VENV_CHECK=1 ./scripts/do.sh init-docker
+          brew update -q && brew bundle -q
+          ./.github/workflows/scripts/docker-setup.sh
```

Comment:
Separating the code from `lib.sh` since it is only used for the CI.

Reply:
That's good.

Reply:
Wait, we do tell people to run it though. https://develop.sentry.dev/environment/#docker-mac-specific Unfortunately searching for `init-docker` didn't work, but I knew this was there. Opened up an issue for that: TODO

Up to you if you want to keep providing test coverage for that or remove that from docs and just have it to facilitate CI. But the work's already done and being tested, so might as well keep it around, right?

Reply:
Also since it's only being run by 1st time setup, we should only be concerned with the latest and greatest, so we shouldn't need to keep that file around right? Whenever CI for this fails, it's more a notification that this stuff needs to be updated.

Reply:
@joshuarli thanks for catching this. I can't believe I forgot!